Makefile

```diff
@@ -185,16 +198,24 @@ ASSET_FILES_OUT := $(foreach f,$(ASSET_FILES_XML:.xml=.c),$f) \
 
 UNDECOMPILED_DATA_DIRS := $(shell find data -type d)
 
+# For now, ROM segments are still taken from the Debug ROM even when building other versions
```

Comment:
Very small nit picky, but could you add a `TODO`s for comments like this. I know they are meant to be short lived and you might even already be working on them, but it just makes them easier to find in the future.

Reply:
Sounds good, done